### ISSUE TITLE
Make "valid-title" check duplicatePrefix with both "it" and "test" interchangeably 

### ISSUE THREAD

**Issue by lukyth** [jest-community/eslint-plugin-jest#742]
Jest has `it` as an alias to `test`, so right now the rule still allow:
```js
test("it should ...", () => {})
// and
it("test should", () => {})
```
which is still pretty redundant.

How about we make the rule check the first word against both "test" and "it"? If so, those two example cases should fail the lint.

If you think this is a good idea, I'm happy to open a PR for it as well.

**Comment by G-Rath** [comment]
I don't think there's a lot wrong with those titles, so I'm hesitant to implement this directly into the rule - you can achieve what you want on your own codebases by using `disallowedWords` and `mustNotMatch`.

The goal of checking if the descriptions start with the function name is that that'll *never* make grammatical sense, but what you're proposing will be affecting descriptions that are perfectly valid English (at least in most cases).

**Comment by lukyth** [comment]
I see. That's understandable. I'll close this then. Thank you so much for the answer! 🙇
